og_sm_path/og_sm_path.admin.inc

```diff
@@ -149,14 +159,15 @@ function _og_sm_path_form_pathauto_patterns_form_alter(array &$form, FormStateIn
   if (!$default_bundle) {
     return;
   }
+
   $node_type = NodeType::load($default_bundle);
   if (!OgSm::siteTypeManager()->isSiteType($node_type)) {
     return;
   }
 
   $form['#disabled'] = TRUE;
-  $args = ['%type' => $node_type->label()];
-  drupal_set_message(
-    t('The %type node type is a Site type, these are aliased using the Site path settings. The path is set in the create/update node form.', $args)
-  );
+
+  Drupal::messenger()->addStatus(t('The %type node type is a Site type, these are aliased using the Site path settings. The path is set in the create/update node form.', [
```

Comment:
TranslatableMarkup maar t werkt ook

Reply:
Waarom geef je de voorkeur om rechtstreeks een een TranslatableMarkup object te maken? Is dat een best practice ofzo?